aries_vcx/README.md

```diff
@@ -1,62 +1,43 @@
-# Aries-vcx
-Crate implementing Hyperledger Aries protocols and building blocks for building Aries agents for
-both mobile (typically in role of holder, prover) and server use-cases (issuer and verifier).
+# aries-vcx
+`aries-vcx` is more of a "library" rather than "framework". We strive to be not too 
+opinionated and simply provide building blocks for whatever you want to build. 
 
-# Getting started
-Aries-vcx is library, not a framework. We strive to be not too opinionated and simply provide building block for whatever
-you want to build. 
+You can use `aries-vcx` to build both end-user products (mobile wallets in role of credential 
+holder and prover) or server (typically in role of issuer, verifier). 
 
-Generally, the crate allows you to 
+`aries-vcx` provides support to:
 - create encrypted wallet, 
 - read/write from/to Indy ledger,
 - establish didcomm connections and exchange messages,
-- create and process Aries messages to drive Aries protocols. 
-
-Have look at [aries-vcx-agent](../agents/rust/aries-vcx-agent) for inspiration how aries-vcx can be used.
+- create and process Aries messages to drive Aries protocols.
 
-# Message mediation
-If you are building mobile agent, you will generally require mediator service, which will receive
-messages on device's behalf - sort of like a mail server.
+## `aries-vcx` components
+If you don't want to use `aries-vcx` directly, you might still find useful smaller components it's
```

Comment:
This is not a very elegant sentence, I would leave at e.g. "The following crates may be consumed independently of `aries-vcx`: ..."